components/patient-data/impl/src/main/resources/ApplicationResources.properties

```diff
@@ -205,6 +206,7 @@ phenotips.UIXField.dateOfBirth=Date of birth
 phenotips.UIXField.dateOfDeath=Date of death
 phenotips.UIXField.lifeStatus=Life status
 phenotips.UIXField.diagnosisNotes=Additional comments
+phenotips.UIXField.geneComments=Gene comments
```

Comment:
I was just making that comment (and others you've already added). `Genetic data notes` sounds best imo (I was going to suggest `Genotype notes`).